visallo/visallo
# RAW Metadata should not be User Visible

If raw metadata property is set, the diff panel is unable to properly display the changes. 

This property is used to hold any RAW metadata that hasn't been explicitly pulled into other properties. As such, it ends up being a JSON object of various fields that aren't all that useful to the user.


Testing Instructions:
- Upload a JPEG image with metadata and wait for the metadata extractor to process it
- Ensure that the diff panel opens properly

CHANGELOG
Changed: Changed `http://visallo.org/minimal#rawMetadata` to not user visible in minimal.owl
